app/assets/javascripts/govuk_publishing_components/analytics-ga4/gtm-click-tracking.js

```diff
@@ -25,7 +25,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
           link_url: window.location.href.substring(window.location.origin.length),
           ui: JSON.parse(target.getAttribute('data-gtm-attributes')) || {}
         }
-        var ariaExpanded = this.getClosestAttribute(target, 'aria-expanded')
+
+        // Ensure it only tracks aria-expanded in an accordion element, instead of in any child of the clicked element
+        if (target.closest('[data-module="govuk-accordion gem-accordion"]') || target.closest('[data-module="gem-accordion govuk-accordion"]')) {
```

Comment:
I haven't come across this before - presumably it was necessary to include both these conditions because the two items in `data-module` could occur in either order? Would a simpler alternative be to look for just one of them in the returned value?

Reply:
@andysellick I tested this on the console on the component guide Accordion, and it looks like querySelectors require an exact match, so just doing `data-module="govuk-accoridon"` doesn't work as you need the full `data-module` value. So I added a check for both orders of the `data-module`. Is this OK?

Reply:
That's fine, although it seems over-complicated - could you search for `target.closest('.gem-c-accordion')` instead?

Reply:
Thanks @andysellick I'm happy to do that - though I was wondering what your thoughts are on coupling the analytics up to classes? I'm guessing that class is very unlikely to change?

Reply:
I think it's very unlikely to change, yes. But you raise a good point - in the event that the classname (or even the data-module attribute) were to change, how would we know? I suppose the analytics would stop working for this expanding element on the accordion - hopefully that would be ultimately picked up by some of the testing that @gclssvglx is looking into?

Reply:
Thanks @andysellick - yep it should be picked up his testing, I can change it to use the class instead :+1:

Reply:
@andysellick I've updated this now :+1: